device-types/Cisco/C8200-1N-4T.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Cisco
+model: Catalyst 8200-1N-4T
+slug: cisco-c8200-1n-4t
+part_number: C8200-1N-4T
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 8200 Series Edge Platforms Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/catalyst-8200-series-edge-platforms/nb-06-cat8200-series-edge-plat-ds-cte-en.html)'
+front_image: true
+rear_image: true
+weight: 10
+weight_unit: lb
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 150
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 150
```

Comment:
I would model this either as a module or as a DC Terminal (or Molex if that is available)

Reply:
added as module